cmd/app/add_test.go

```diff
@@ -484,7 +484,7 @@ func prepareAddMocks(t *testing.T, clients *shared.ClientFactory, clientsMock *s
 		Return("logstash host")
 
 	manifestMock := &app.ManifestMockObject{}
-	manifestMock.On("GetManifestLocal", mock.Anything, mock.Anything).Return(types.SlackYaml{
+	manifestMock.On("GetManifestLocal", mock.Anything, mock.Anything, mock.Anything).Return(types.SlackYaml{
```

Comment:
note: A common change is updating mocks to support the added `ctx context.Context` argument.